packages/adobe_analytics/ios/Classes/SwiftAdobeAnalyticsPlugin.swift

```diff
@@ -73,11 +73,15 @@ extension SwiftAdobeAnalyticsPlugin {
 extension SwiftAdobeAnalyticsPlugin: AdobeAnalyticsProtocol {
   
   public func trackAction(_ action: String, data: [String : String]?) {
-    ACPCore.trackAction(action, data: data)
+    if (UIApplication.shared.applicationState != .background) {
```

Comment:
Perhaps it's safer to do it on `== .active`, otherwise this might go trough in the `.inactive` state:

> The app is running in the foreground but is not receiving events. This might happen as a result of an interruption or because the **app is transitioning to or from the background.**

Reply:
Adobe only considers it a crash if a tracking happened after `UIApplicationDidEnterBackgroundNotification`.
So tracking during inactive is still fine I think.
https://aep-sdks.gitbook.io/docs/using-mobile-extensions/mobile-core/lifecycle/lifecycle-extension-in-ios